### PR TITLE
Removes the Psychiatrist

### PR DESCRIPTION
**Pull request by swissloaf**
# General Documentation

### Intent of your Pull Request

The purpose of this PR is that it removes the psychiatrist (the job) and his office from the game. The psychiatrist's traitor tools (hypno flash and cat grenade) have all been moved to med doctor-exclusive items. The psych office itself has been modified in Boxstation, Icebox, Meta, Eclipse, and Delta. They have all been removed from those maps EXCEPT in Eclipse, where it is now the "General Evaluation" room. 

**I've made it an empty space (with some modifications so it doesnt look like complete shit) so that a mapper may come along in the future and put something worthwhile there or edit it to how they see fit.**

### Why is this change good for the game?

The psychiatrist is never played, and as discussed in quickfire and admin chats, when it is played it is played as a "assistant with medbay access". Very rarely has proper roleplay come out of this position, and there seems to be no way to make the Psychiatrist position fun for the players nor the person playing the role. The position itself is better off being removed before it causes any more trouble. This vote is based off the result in #quickfire-votes. If you want to see more reasoning behind this, check out the discussion in #quickfire-vote-discussion.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
The Psychiatrist role is removed, and the maps are edited to exclude the psych office except in Eclipse where it has been renamed.

### What should players be aware of when it comes to the changes your PR is implementing?

They are losing a job position, though they never really played it.

### What general grouping does this PR fall under? 
Medical changes - job being removed? 
Map changes

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
For example, "This new chem will heal X brute damage". 

# Changelog

:cl:  
rscdel: Removes the Psychiatrist
/:cl:

### CODE REVIEW

**Review by jcatjmeow — APPROVED** [review]

**Comment by PrinceKirze** [comment]
rest in peace
pour one out for a real one

**Comment by Hopekz** [comment]
I like the Psych and have played many hours as a Psych, so I can't entirely agree that it doesn't add anything to the game.
Instead of removing Psych modify it to fit with what you think it should do.
It is a great RP role IMO and always has been. 
What's next are you going to remove the Clerk and add an empty room instead?

**Comment by swissloaf** [comment]
> I like the Psych and have played many hours as a Psych, so I can't entirely agree that it doesn't add anything to the game.
> Instead of removing Psych modify it to fit with what you think it should do.
> It is a great RP role IMO and always has been.
> What's next are you going to remove the Clerk and add an empty room instead?

the clerk is actually useful and a good role sometimes, which is surprising because its not even a department-based role lmao

**Comment by swissloaf** [comment]
to clarify, people actually interact with the clerk in a rp setting, and even if it isnt rp, they buy from him. nobody wants to rp with the psych and do boring "psych evaluation rp". its just boring

**Comment by LazennG** [comment]
Don't think psych would be seen as dead weight if the role was able to do stuff focused around mood and traumas like boost mood a bit after spending some time with the psych or proc undoing minor traumas after a few minutes and calling it 'rehab' but what do i know about coding

**Comment by UselessTheremin** [comment]
psych boils down to, in my experience:
"crazy drug addict RP" with self harm / toeing the line with grief rules, often resulting in bwoinks.
"psych evaluations" which is literally just either paperwork or putting someone in a straightjacket for a whole round because they were acting weird and you were bored.
and finally, the most consistent one, just literally being an assistant with access to medbay. psych is never going to be truly gone because you can literally roll assistant and do that as a gimmick, and in my mind i feel it is better as that.

**Comment by Tipy1802** [comment]
Literally why, other than “me not like playing psych and I had one bad interaction”. What Theremin said is not even that common. Can we remove atmos tech? I don’t like playing it and they often end up griefing the distro or blowing shit up. 

 Like come on roles like assistant, clown, atmos tech and miner toe and break the rules far more than psychiatrist.

Can we also get pictures of what the psychiatrist office has been turned into, better it be something good to warrant the removal of a whole job

**Comment by PrinceKirze** [comment]
i agree it'd be nice to get pictures of the new psych room replacement

**Comment by MegaEmpirical** [comment]
There's no reason to remove psych, it existing hurts no one and only provides an additional option for people to choose from.

As for the room, it can be removed from medbay, that's fine. The psych could get a new office in the arrivals hall really easily as there is the old commissary and the old clerk room, and even the vacant office. Any of those could easily be repurposed and would give the arrival hall a little extra something which I think is nice while also giving people greater access to the psych and vice versa which I think is useful for a high RP role.

**Comment by Readystorm546** [comment]
🦀

**Comment by swissloaf** [comment]
BoxStation
image
(Same on IceBox with few modifications, namely removal of the trimming)

DeltaStation
image

Meta
image

EclipseStation stays the same

**This PR doesn't change it to a different room or add a new one, it simply creates a space where a mapper can come and make something out of it for the future, which I expect would happen expediently afterwards if this PR is merged** You shouldn't upvote nor downvote based upon the map changes because the focus of this PR is to simply remove the office, not make it into a new room. That's for people with an idea for what it should be to come in and do.

**Comment by Moja10** [comment]
When I made the quickfire question, I was afraid of a PR instantly just jumping on the results and not even considering how many people voted to keep it, and ignoring how many people want to see it reworked. I would prefer to pursue opportunities and ideas to rework Psych before considering removal, or exploring other options like enabling moods on everyone, creating new drugs for the psyche to use, or adding some actual functionality to the role rather than this PR. -1 

If after a good while no good rework or new ideas or features get added, then we can discuss removal but I think this is a bit hasty.

**Comment by Ratvar** [comment]
All these years job existed withot an update were the good while, it already happened
Too many people that voted to keep it are scared of anything with word removal

**Comment by swissloaf** [comment]
I feel like people can't actually read vote results. People want it to be reworked, of course, (if it cant be removed) because its dogshit in its current state. Along with this, there have been no real plans to rework it that people have said they will "sit down and do it". We can talk talk talk about how we should rework it but unless there's an actual plan to go ahead and code it, no. Besides, over a majority of the people want it just removed in its entirety.

**Comment by redmoogle** [comment]
@swissloaf please pull from upstream/master when you can so that your PR can compile. feel free to ask how in #development-public if you don't know how

**Comment by MegaEmpirical** [comment]
A simple majority is not good reason in of itself to remove something.

**Comment by redmoogle** [comment]
> A simple majority is not good reason in of itself to remove something. 

Well some other reason I see ingame myself

Psych has mostly been unfilled. So at this point its became wasted space which can be used to make something. There's also some antag gear blocked behind psych so now MDs can use them now.

**Comment by MegaEmpirical** [comment]
> > A simple majority is not good reason in of itself to remove something.
> 
> Well some other reason I see ingame myself
> 
> Psych has mostly been unfilled. So at this point its became wasted space which can be used to make something. There's also some antag gear blocked behind psych so now MDs can use them now.

See my initial post.

**Comment by patpol4** [comment]
I don't see why would we do this as much as I don't see why wouldn't we do this

**Comment by redmoogle** [comment]
> > > A simple majority is not good reason in of itself to remove something.
> > 
> > Well some other reason I see ingame myself
> > 
> > Psych has mostly been unfilled. So at this point its became wasted space which can be used to make something. There's also some antag gear blocked behind psych so now MDs can use them now.
> 
> See my initial post. 

Problem is that I dont think anyone will ever play yogs for RP

**Comment by MegaEmpirical** [comment]
Regardless if they do or not those rooms are effedtibely unsued space and the psych existing doesn't hurt. Moving the office out of med frees that space and spruced up the station a little.

**Comment by swissloaf** [comment]
> Regardless if they do or not those rooms are effedtibely unsued space and the psych existing doesn't hurt. Moving the office out of med frees that space and spruced up the station a little.

yeah, but psych is a medical role and should be in medbay. they literally dispense pills

**Comment by Georgeur** [comment]
only reason this is necessary is because we removed dabbing which soft nerfed psych as a lot less people got brain traumas. 
add dabbing back and close the pr please

**Comment by MegaEmpirical** [comment]
Their office doesn't need to be in medical at all and if that's really such an important point then rename them as psychologist and put them under the service department instead.

**Comment by swissloaf** [comment]
> Their office doesn't need to be in medical at all and if that's really such an important point then rename them as psychologist and put them under the service department instead.

great we'll have an even more useless role. now they cant even dispense medicine

**Comment by Takahiru** [comment]
but i like the antag item

**Comment by Ratvar** [comment]
Agree, make psych antag item into md item, more people having fun with it while having more tools is good

**Comment by swissloaf** [comment]
> Agree, make psych antag item into md item, more people having fun with it while having more tools is good

That exactly is what it says in the PR description

**Comment by Ratvar** [comment]
> That exactly is what it says in the PR description

This is the point Taka did not get, ye

**Review by patpol4 — APPROVED** [review]
Leaving empty spaces like this just leaves the medbay empty, what I did is add the offices back but renamed them as "Evaluation Office".
The spawn and locker is gone but the office stays allowing for some antag play or medical inspection rp behind closed shutters.
I don't have any strong feelings on the removal one way or the other, up to Jamie to have the final say.

**Comment by SomeguyManperson** [comment]
@yogstation13/head-coders @yogstation13/council  are we gonna remove psych or rework it or what the changes in this pr ard holding up any other map changes so we need a decision
